plugins/promela/hu.bme.mit.gamma.promela.verification/src/hu/bme/mit/gamma/promela/verification/PromelaVerification.xtend

```diff
@@ -34,10 +34,13 @@ class PromelaVerification extends AbstractVerification {
 	}
 	
 	override getDefaultArguments() {
-		return #[ "-search -a" ]
+		return #[
+			 "-search -a -b" // default
+//			 "-search -a -m10000000 -DVECTORSZ=4096" // recommended for models with ParallelAction			 
```

Comment:
Cannot we find a default argument that works well both for sequential and parallel programs?